Testing/SystemTests/lib/systemtests/systemtesting.py

```diff
@@ -12,6 +12,11 @@
 File change history is stored at: <https://github.com/mantidproject/systemtests>.
 """
 from __future__ import (absolute_import, division, print_function)
+# == for testing conda build of mantid-framework ==========
+import os
+if os.environ.get('MANTID_FRAMEWORK_CONDA_SYSTEMTEST'):
+    import matplotlib
```

Comment:
Could I ask why `matplotlib` is required here?

Reply:
It seems it has been a while that with conda mantid-framework, one has to import matplotlib before mantid to avoid a segfault.

Reply:
Could you add a comment about that? We should investigate that too...